diff --git a/browser/base/content/browser.js b/browser/base/content/browser.js
index f59f732..07a9a60 100644
--- a/browser/base/content/browser.js
+++ b/browser/base/content/browser.js
@@ -5445,8 +5445,6 @@ var XULBrowserWindow = {
 
     SafeBrowsingNotificationBox.onLocationChange(aLocationURI);
 
-    SaveToPocket.onLocationChange(window);
-
     UrlbarProviderSearchTips.onLocationChange(
       window,
       aLocationURI,
diff --git a/browser/components/BrowserGlue.jsm b/browser/components/BrowserGlue.jsm
index 64ac529..8cf69c1 100644
--- a/browser/components/BrowserGlue.jsm
+++ b/browser/components/BrowserGlue.jsm
@@ -1269,7 +1269,6 @@ BrowserGlue.prototype = {
       lazy.Normandy.init();
     }
 
-    lazy.SaveToPocket.init();
 
     AboutHomeStartupCache.init();
 
diff --git a/browser/components/moz.build b/browser/components/moz.build
index 71a1b85..ed3b6d1 100644
--- a/browser/components/moz.build
+++ b/browser/components/moz.build
@@ -42,7 +42,6 @@ DIRS += [
     "originattributes",
     "ion",
     "places",
-    "pocket",
     "preferences",
     "privatebrowsing",
     "prompts",
